packages/visual-engine/tests/adaptHtmlElement.spec.ts

```diff
@@ -31,8 +31,8 @@ describe('visual engine tests', () => {
     adaptHtmlElement(htmlElement, settings, 'jsdom')
 
     const readaptStyles = document.head.querySelector('[data-readapt-style=jsdom]') as HTMLStyleElement
-    expect(readaptStyles.textContent).toBe(
-      '.readapt-jsdom .readapt-content { font-size: inherit; } .readapt-content > span { font-size: inherit } .readapt-jsdom .readapt-bold { font-weight: bold }'
+    expect(readaptStyles.textContent?.replace(/\n/g, ' ').replace(/ +/g, ' ').trim()).toBe(
+      '.readapt-jsdom .readapt-content { font-size: inherit; } .readapt-jsdom .readapt-content > span { font-size: inherit } .readapt-jsdom .readapt-content .readapt-bold { font-weight: bold }'
```

Comment:
```suggestion
    expect(readaptStyles.textContent).toContain(`
    .readapt-jsdom .readapt-content { font-size: inherit; }
    .readapt-jsdom .readapt-content > span { font-size: inherit }
    .readapt-jsdom .readapt-content .readapt-bold { font-weight: bold }
  `)
```

I'd say to use exactly the same string since it's easier to read.

Alternatively, we could use [.toEqualIgnoringWhitespace(string)](https://jest-extended.jestcommunity.dev/docs/matchers/string/#toequalignoringwhitespacestring) from `jest-extend` package.